lib/linux_admin/network_interface/network_interface_rh.rb

```diff
@@ -162,7 +162,9 @@ def apply_static6(ip, prefix, gw, dns, search = nil)
     def save
       old_contents = File.read(@interface_file)
 
-      return false unless stop
+      stop_success = stop
+      stop_success = stop unless stop_success
+      return false unless stop_success
```

Comment:
So I opened a BZ that describes why we have to do this (https://bugzilla.redhat.com/show_bug.cgi?id=1472396)

Can you add a comment here with the BZ link explaining why we are trying twice?